portal/src/graphql/portal/SubscriptionScreen.tsx

```diff
@@ -608,25 +608,23 @@ function SubscriptionScreenContent(props: SubscriptionScreenContentProps) {
   const [enterpriseDialogHidden, setEnterpriseDialogHidden] = useState(true);
   const [cancelDialogHidden, setCancelDialogHidden] = useState(true);
 
-  // @ts-expect-error
```

Comment:
I prefer using ts-expect-error to casting to unknown. In this particular case, the problem is the type definition is too strict on the type of `subText`. In case the type definition is corrected in a future version of fluent UI, the ts-expect-error will become invalid. This will be caught by tsc. If we cast to unknown, then the compiler will not be able to do its job.

Reply:
Oh I see. I changed this because I think the `@ts-expect-error` here ignored all errors in the whole object, but what we only have type error in `subText` line. Maybe I rewrite it so that we only ignore type errors in `subText`.